README.md

```diff
@@ -84,6 +84,29 @@ $ curl -X GET http://localhost:8008/v1
 into, run `nexus` with `CONSENSUS_DAMASK_GENESIS_DUMP=/tmp/genesis.sql`. The SQL will be written to the indicated
 file, provided that genesis hasn't been parsed into the DB yet. The easiest way to achieve the latter is to wipe the DB.
 
+### Code quality tools
+
+Here are our recommendations for getting the tools that `make lint` and `make fmt` use.
+
+**golangci-lint**: [use their `curl | sh` installer](https://golangci-lint.run/usage/install/).
+They explain that the tool is in dependency hell and any more structured distribution of it might not work.
+
+```sh
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
```

Comment:
```suggestion
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.49.0
```

to match CI

Reply:
hm I don't think we can maintain a copy of the version number here

Reply:
o crud I'm actually not even on that version locally

Reply:
eh I'll add it